README.md

```diff
@@ -80,34 +96,35 @@ mkdir -p /tmp/rstudio-server
 
 Then launch the server similar to the docker command but with additional configuration options required:
 
+Set a password for your RStudio server:
+```
+RSERVER_PASSWORD=$(openssl rand -base64 15)
+echo $RSERVER_PASSWORD
+```
+Please notedown this RSERVER_PASSWORD as you will need when you log on to RStudio from a browser.
```

Comment:
Fix typo here and reword for clarity. I suggest: 

```
Take note of the password returned by your echo command. You will need this to log into RStudio in your browser. 
```

Reply:
DONE